lib/kast.js

```diff
@@ -293,8 +293,7 @@ const format = (o, isRaw = false, mixFix = false, colorize = false) => {
       "_-Int_"             : " - "   ,
       "_*Int_"             : " * "   ,
       "_/Int_"             : " / "   ,
-      "_%Int_"             : " % "   ,
-      "_modInt_"           : " % "   ,
+      "_modInt_"           : " mod " ,
```

Comment:
I like `%` more than `mod`, but no strong feelings.

Reply:
`%` looks nicer, but I think it's lying a bit. `K` has both a `%Int` and a `modInt` with slightly different semantics (see [here](https://github.com/kframework/k/blob/master/k-distribution/include/builtin/domains.k#L379) and [here](https://github.com/kframework/kore/blob/master/docs/hooks.md#inttmod)). `modInt` is the one that is used in the `evm-semantics`, and I think it's good if the `klab` debugger faithfully reflects that.

Reply:
They mean different things, btw. `%Int` and `modInt` in K have different implementations. `-5 modInt 3 =/= -5 %Int 3`. One gives back `-2`, the other `1`. I can't remember which is which, but KEVM specifically chooses the right ones.